internal/command/command.go

```diff
@@ -18,7 +19,7 @@ type Command struct {
 func New() *Command {
 	flags, err := config.ParseFlags()
 	if err != nil {
-		utils.Fatal(err)
+		os.Exit(2)
```

Comment:
Why exit code 2? If you want to have a different exit code for every exit reason, I think it would make sense to have all of them as an enum/iota const. But I'm not sure if that's even worth the effort.

Reply:
Bad CLI args = exit code 2. We shouldn’t re-invent this wheel.

Reply:
Does it? [citation needed]

Reply:
https://github.com/golang/go/blob/1f9c9d853067635305f72e247c5b49e3fa5da8af/src/flag/flag.go#L317

Reply:
The current implementation does not follow that convention as it also exits with code 2 when the help text is printed. Anyways, this is only the behavior of the Go flag package, not some standard (or even common AFAICT) behavior, so I'd say it's valid to discuss this.

I'd at the very least add a comment that this follows the exit code convention of the Go flag package (if it actually does), or simply go with exit code 1 as probably nobody will check this exit code anyways.